src/ssl.c

```diff
@@ -13272,6 +13309,125 @@ long wolfSSL_CTX_sess_number(WOLFSSL_CTX* ctx)
 }
 
 #ifndef NO_DES3
+void wolfSSL_DES_set_key(WOLFSSL_const_DES_cblock* myDes,
+                                               WOLFSSL_DES_key_schedule* key)
+{
+#ifdef WOLFSSL_CHECK_DESKEY
+    wolfSSL_DES_set_key_checked(myDes, key);
+#else
+    wolfSSL_DES_set_key_unchecked(myDes, key);
+#endif
+}
+
+
+
+/* return true in fail case (1) */
+static int DES_check(word32 mask, word32 mask2, unsigned char* key)
+{
+    if ((*(word32*)(key + 0) ^ mask) == 0) {
```

Comment:
Can we really assume key is 32bit aligned?  On some platforms if this isn't the case this will fault.